code/datums/components/omen.dm

```diff
@@ -8,25 +8,33 @@
  */
 /datum/component/omen
 	dupe_mode = COMPONENT_DUPE_UNIQUE
-
 	/// Whatever's causing the omen, if there is one. Destroying the vessel won't stop the omen, but we destroy the vessel (if one exists) upon the omen ending
 	var/obj/vessel
-
 	/// Whether this is a permanent omen that cannot be removed by any non-admin means.
 	var/permanent = FALSE
-
-/datum/component/omen/Initialize(silent = FALSE, _vessel, _permanent = FALSE)
+	/// Whether this was caused by a quirk
+	var/quirk = FALSE
+	/// The outer light range of the self-gib explosion
+	var/explode_outer = 0.8
+	/// The force of the self-gib explosion
+	var/explode_inner = 0
+	/// Luck modifier. Higher means more likely to trigger, more damage, etc. Cursed only half as unlucky
+	var/luck_mod = 1
+
+/datum/component/omen/Initialize(silent = FALSE, _vessel, _permanent = FALSE, _quirk = FALSE)
 	if(!isliving(parent))
 		return COMPONENT_INCOMPATIBLE
 	vessel = _vessel
 	permanent = _permanent
+	if(_quirk)
+		quirk = _quirk
+		luck_mod = 0.5 // Lowers damage and probabilities for naturally cursed
```

Comment:
this should be an arg too

Reply:
same with explosion radius

Reply:
also i think the luck mod could be even lower, falling down is still something like 37 damage

Reply:
yeah, i agree. the original omen is nearly a death sentence. 75ish and 100 brain dmg? ouch. 
what if we changed _quirk to make it 50% probability and 25% damage?
i wanted the ability for admins to change the probabilities and damage in VV, as long as they can still do this as an arg, ok

Reply:
change these names to be `src.quirk = quirk` and remove the underscore from the parameter names